modin/pandas/base.py

```diff
@@ -1387,6 +1387,7 @@ def drop_duplicates(
         """
         inplace = validate_bool_kwarg(inplace, "inplace")
         subset = kwargs.get("subset", None)
+        ignore_index = kwargs.get("ignore_index", False)
```

Comment:
is it preferable to pull from `kwargs` or to add an argument and default value to the function?